dlc/src/main/scala/org/bitcoins/dlc/DLCTimeouts.scala

```diff
@@ -10,4 +13,24 @@ case class DLCTimeouts(
     penaltyTimeout: Int,
     contractMaturity: BlockStampWithFuture,
     contractTimeout: BlockStampWithFuture
-)
+) extends NetworkElement {
+  override def bytes: ByteVector = {
+    ByteVector.fromInt(penaltyTimeout) ++ contractMaturity.toUInt32.bytes ++ contractTimeout.toUInt32.bytes
```

Comment:
I think the `penaltyTimeout` should be encoded as a `UInt32` as well as is convention for on-chain timeout numbers